packages/fuels/src/cli.ts

```diff
@@ -90,10 +90,12 @@ export const configureCli = () => {
    * Routing external commands from sub-packages' CLIs
    */
 
-  // Typegen
-  configureTypegenCliOptions(
-    program.command('typegen').description(`Generate Typescript from Sway ABI JSON files`)
-  );
+  (command = program.command(Commands.typegen))
```

Comment:
Yes, they're not necessary anymore because the abis now contain the program type. https://github.com/FuelLabs/fuels-ts/pull/3249/commits/1703d8fd754271c4b200452a6fc2c27bd07991f0